release/models/qos/openconfig-qos-elements.yang

```diff
@@ -1231,6 +1237,18 @@ submodule openconfig-qos-elements {
             term - such that packets that arrive in the queue are
             immediately serviced.";
         }
+        enum DWRR {
+          description
+            "This scheduler term is considered as a deficit weighted
+            round robin term - such that packets that arrive in the queue are
+            serviced based on deficit counter with weighted round robin fashion.";
+        }
+        enum WRR {
+          description
+            "This scheduler term is considered as a weighted
+            round robin term - such that packets that arrive in the queue are
+            serviced based on weighted round robin fashion.";
+        }
       }
```

Comment:
Are there any strong references that provide a strong definition of what WRR or DWRR actually means?  My fear is that at least DWRR may end up being quite platform/hardware specific and if that is the case I'm thinking that we may need to pull that out in the description.

Reply:
+1. 

opening a featureprofiles PR with a functional test README would also help clarify the behaviour of these new enum values. 

For examples, see the following tests:
* https://github.com/openconfig/featureprofiles/tree/main/feature/qos/otg_tests/egress_strict_priority_scheduler_test
* https://github.com/openconfig/featureprofiles/tree/main/feature/qos/otg_tests/egress_strict_priority_scheduler_with_bursty_traffic_test

Reply:
Aruba seems to also use the DWRR term: https://arubanetworking.hpe.com/techdocs/AOS-CX/AOSCX-CLI-Bank/cli_832x/Content/QoS_cmds/dwrr-queue.htm